knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml

```diff
@@ -1364,7 +1364,7 @@ data:
         ]
       },
       "timezone": "",
-      "title": "Knative Eventing - Broker/Trigger",
+      "title": "Knative Eventing - Channel-based Broker/Trigger",
```

Comment:
This requires changes in the docs, for example https://docs.openshift.com/container-platform/4.9/serverless/admin_guide/serverless-admin-monitoring.html
```
Select the Knative Eventing - Broker/Trigger dashboard in the Dashboard drop-down list.
```
Let's open a jira for this too.

Reply:
Let me capture a Jira and I will work on it next (if the changed title makes sense to you).

Reply:
https://issues.redhat.com/browse/SRVKE-1173